src/index.tsx

```diff
@@ -785,7 +788,6 @@ export class App extends React.Component<any, AppState> {
                 lastMouseUp = null;
                 isPanning = false;
                 isHoldingMouseButton = false;
-                if (!isHoldingSpace) resetCursor();
```

Comment:
not sure if this removal was intentional, but it's necessary so I reintroduced it.

Reply:
The problem is that it's causing the default cursor to appear instead of the crosshair one after panning when a shape tool is selected.

Reply:
When you wheel+drag to pan, and you have a different tool other than selection. Haven't dug into it further.